src/main/java/pl/sda/javapoz/security/SecurityConfig.java

```diff
@@ -3,47 +3,53 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
 import org.springframework.social.connect.ConnectionFactoryLocator;
 import org.springframework.social.connect.UsersConnectionRepository;
 import org.springframework.social.connect.mem.InMemoryUsersConnectionRepository;
 import org.springframework.social.connect.web.ProviderSignInController;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import pl.sda.javapoz.service.CustomUserDetailsService;
 import pl.sda.javapoz.service.FacebookConnectionSignup;
 import pl.sda.javapoz.service.UserService;
 
-/**
- * Created by pablo on 23.03.17.
- */
+
 @Configuration
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
     @Autowired
     private ConnectionFactoryLocator connectionFactoryLocator;
 
     @Autowired
+
     private UsersConnectionRepository usersConnectionRepository;
 
     @Autowired
-    private FacebookConnectionSignup facebookConnectionSignup;;
+    private FacebookConnectionSignup facebookConnectionSignup;
 
     @Autowired
     private UserService userService;
 
+/*
```

Comment:
co to jest ?

Reply:
w sensie ... czy ten konstruktor jest wymagany w security? :)

Reply:
No wlasnie sie zastanawialem, ale w sumie dlaczego nie, jak to jest normalna klasa przeciez, wiec raczej powinno sie zrobic tak jak inne.